src/main/java/org/gridsuite/network/map/dto/bus/AbstractBusInfos.java

```diff
@@ -0,0 +1,20 @@
+package org.gridsuite.network.map.dto.bus;
```

Comment:
You need to add copyright:
/**
 * Copyright (c) 2023, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */